.github/workflows/scheduled-build.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Scheduled Build'
+on:
+  schedule:
+    - cron: '11 1 * * SAT'
+
+jobs:
+  scheduledBuild:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        branch: [master, release]
```

Comment:
Do the steps include cleanup, or are matrixed builds separated? (As to prevent master and release repos getting mixed by using the same working dir.)

Reply:
They are completely separated into different runs (and thus both start with a clean VM)